database/schema.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+
+const schema = new mongoose.Schema({
+  title: String,
+  city: String,
+  price: Number,
+  msrp: Number,
+  dates: [Date],
```

Comment:
does [Date] mean it will be an array of various dates e.g. 12/24, 12/25, 12/26, etc.?  assuming so

Reply:
yea, so far i just have 1 date in each of the arrays

Reply:
cool yeah in an ideal situation, we'd eventually have it so the number of dates on the calendar matches up with however many days the main page says the trip lasts